engine/env/spark/src/main/java/org/datacleaner/spark/functions/RowProcessingFunction.java

```diff
@@ -69,8 +71,8 @@
  * This class implements two interfaces because it has two (quite similar)
  * styles of usages in the {@link SparkAnalysisRunner}.
  */
-public final class RowProcessingFunction implements
-        Function2<Integer, Iterator<InputRow>, Iterator<Tuple2<String, NamedAnalyzerResult>>>,
+public final class RowProcessingFunction
+        implements Function2<Integer, Iterator<InputRow>, Iterator<Tuple2<String, NamedAnalyzerResult>>>,
         PairFlatMapFunction<Iterator<InputRow>, String, NamedAnalyzerResult> {
```

Comment:
It's pretty strange - I have imported the ```Formatter-Eclipse.xml``` file. It does it like this on my Mac, but I think (given experience) that it will be OK on my Windows machine.